.github/workflows/socialsportsprod_cloud_run_deploy_ci.yml

```diff
@@ -0,0 +1,155 @@
+name: Deploy cloud functions to Cloud Run and Firebase
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
```

Comment:
This here is for development purposes so the pipeline can be tested each time a change is made to this pull request. Once the changes have been approved and ready to merge, I will remove this on trigger.